requirements/base.txt

```diff
@@ -32,15 +32,15 @@ pygments==2.13.0
     # via sphinx
 pyparsing==3.0.9
     # via packaging
-pytz==2022.4
+pytz==2022.5
     # via babel
 requests==2.28.1
     # via sphinx
 six==1.16.0
     # via edx-sphinx-theme
 snowballstemmer==2.2.0
     # via sphinx
-sphinx==5.2.3
+sphinx==5.3.0
```

Comment:
No breaking change in the new version. For details see Changelog https://www.sphinx-doc.org/en/master/changes.html#release-5-3-0-released-oct-16-2022.